report-flat/src/App.vue

```diff
@@ -0,0 +1,70 @@
+<template>
```

Comment:
Is picking Vue for this app good approach from maintainability perspective - I don't many people in Cognifide have experience with Vue?

What are considerations behind picking Vue as framework for this application instead of using Angular? Is the code easier? More appealing?

Now we have two different frameworks with different paradigms behind in single in single repo - should this app be extracted to separate repo as it's separate app?